api/Document.json

```diff
@@ -1066,6 +1066,130 @@
           }
         }
       },
+      "canplay_event": {
```

Comment:
There's already a "canplay_event" entry for `HTMLMediaElement` and that's the only one we should have, as the event doesn't bubble. Can you revert the other additions, and then compare the data being removed to that existing entry, noting any discrepancies?